addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2331,6 +2346,50 @@ export class OdooEditor extends EventTarget {
                 }
             }
         } else if (node.nodeType !== Node.TEXT_NODE) {
+            if (node.nodeName === 'TD' && node.hasAttribute('style')) {
+                // TD tags do not support style in v15, move style to children or new SPAN.
+                const spans = node.childNodes.length === 0 || [...node.childNodes].some(n => !['SPAN', 'FONT'].includes(n.nodeName))
+                    ? [this.document.createElement('SPAN')]
+                    : [...node.childNodes];
+                node.style.backgroundColor = null; // background-color: on TD (cell) != on span (text).
```

Comment:
Even with the comment, it's not obvious why you then _remove_ it altogether. (I know why, but I don't think an external reader will get it .)

Reply:
> // Background-color on TD (cell) != on span (text).
> // Prevent it from being copied from cell to text.
I wanted to explain why it's safe to remove it but there's a comment just above that says it already
> // TD tags do not support style in v15

Reply:
Yes that would be enough :)